src/users_bar/users_bar.css

```diff
@@ -0,0 +1,17 @@
+.users-bar {
+    min-width: 20%;
+    box-shadow: 1px 0 5px grey;
+}
+
+.users-bar__header {
+    background-color: rgb(0, 188, 212);
```

Comment:
Rename all `.css` to `.sass`/`scss`. and move  cvolor to const

Reply:
fixed